lib/reversetunnel/fake.go

```diff
@@ -70,7 +70,7 @@ type FakeRemoteSite struct {
 func NewFakeRemoteSite(clusterName string, accessPoint auth.RemoteProxyAccessPoint) *FakeRemoteSite {
 	return &FakeRemoteSite{
 		Name:        clusterName,
-		connCh:      make(chan net.Conn),
+		connCh:      make(chan net.Conn, 100000),
```

Comment:
the other one got bumped up to ten thousand, did you mean to put one hundred thousand here? Seems high 🤷‍♂️